api/graphql/schema.graphql

```diff
@@ -815,6 +823,7 @@ type Query {
     includeIds: [UUID!] @in(key: "id")
     excludeIds: [UUID!] @notIn(key: "id")
     where: PoolFilterInput
+    orderByPoolBookmarks: PoolBookmarksOrderByInput @scope
```

Comment:
Why is a second query fired when the orderby is already satisfied in the first query?
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/a3d1d7b5-797b-4e25-ba0a-39333161cccb)

Reply:
I wasn't seeing this, but let me double check.

Reply:
I'm still seeing it but if it's not obvious to you I think we can let it go for later optimization.